Participate/Asia-Pacific-community-call.md

```diff
@@ -0,0 +1,7 @@
+
+Asia Pacific Community Call
+
+This working group aims to connect with community members in the Asia Pacific region regarding open source community health. Discussions will focus on recaping CHAOSS work as well as identifying new areas of interest. The working group is, of course, open to everyone from any global region. 
+
+The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 15:00 CET, check your local time) via Zoom -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
```

Comment:
Please turn "check your local time" into a link to https://arewemeetingyet.com with the meeting times setup